exercises/F-predicates/exercise.js

```diff
@@ -6,11 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0 === true;
+ }
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
-
+function isBetweenZeroAnd10(number) {
+  return number > 0 && number < 10 === true;
+}
 /* 
```

Comment:
No need to check if the return expression "=== true", the expression will evaluate to true or false when it is returned.